data/adoptions.yaml

```diff
@@ -1,20 +1,18 @@
 main:
-- url: https://github.com/awslabs/tough
+- url: https://github.com/cnabio/cnab-spec/blob/cnab-security-1.0.0-ga/300-CNAB-security.md
```

Comment:
We need to add a logo image to `"static/img/logos/adoptions" + lowercase(tag) + "_logo." + img_type | "png"`.

Maybe we can use this one? 
https://user-images.githubusercontent.com/686194/61752644-54580b80-ad61-11e9-9518-608534d09bdd.png